drivers/sensor/veaa_x_3/Kconfig

```
@@ -0,0 +1,18 @@
+# VEAA-X-3 configuration options
+
+# Copyright (c) 2024, Vitrolife A/S
+# SPDX-License-Identifier: Apache-2.0
+
+config VEAA_X_3
+	bool "VEAA-X-3 pressure driver"
```

nit, missing a word:

```suggestion
	bool "VEAA-X-3 pressure regulator driver"
```